### ISSUE TITLE
xpra 5.0-r32148-1 cannot connect via ssh

### ISSUE THREAD

**Issue by allo-** [Xpra-org/xpra#3662]
**Describe the bug**
xpra `5.0-r32148-1` cannot connect via ssh anymore.
Downgrading to version `5.0-r32078-1` fixes the issue for me, so the change must be a regeression between these two version.

xpra probably confuses the local and the remote username.

**To Reproduce**
1. server command

    `remoteuser@remotehost$ xpra start :10`

3. client command

    `localuser@localhost$ xpra attach --ssh=/usr/bin/ssh ssh:remoteuser@server:10`

5. specific action to trigger the bug

Trying to connect with the client

**System Information (please complete the following information):**
 - Server OS: Debian stable
 - Client OS: Debian testing
 - Xpra Server Version 4.4-r0-1
 - Xpra Client Version 5.0-r32148-1

**Additional context**
```
2022-10-21 16:02:21,892 Error: SSH connection to the xpra server failed
2022-10-21 16:02:21,892  check your username, hostname, display number, firewall, etc
2022-10-21 16:02:21,892  for server: ssh://remoteuser@server:10
2022-10-21 16:02:21,893  the command line used was:
2022-10-21 16:02:21,893  /usr/bin/ssh -l localuser -p 10 -T server
```

The command line should contain `-l remoteuser` as parameter or `remoteuser@server` as connection string.

**Comment by totaam** [comment]
Very likely 18720c1f1d9fcade877c6174c7ef65d6580471ea or 998b9c5230dc6a517975f26330027cd621d52db9.
As a temporary workaround, you can use `--ssh=ssh -l remoteuser`.

**Comment by totaam** [comment]
Fixed in 3e05f9a9cabbc135702788281c24ab37eeb9e577

FYI: the syntax `ssh:remoteuser@server:10` is deprecated and may be removed in the future.
Please switch to a more standard URL syntax (#3599): `ssh://user@host:port/`

**Comment by allo-** [comment]
That's, that was a fast fix. I'll update when there are new debian packages. If the error persists, I'll reopen, otherwise you can assume it works for me again.
